cl/people_db/lookup_utils.py

```diff
@@ -328,7 +328,8 @@ def find_just_name(text: str) -> str:
 
     # Next up is full names followed by a comma
     match_titles = re.search(
-        "(((Van|VAN|De|DE|Da|DA)\s)?[A-Z][\w\-'']{2,}(\s(IV|I|II|III|V|Jr\.|JR\.|Sr\.|SR\.))?),",
+        # adding 'r' prefix to address python 3.12 strickness around escape sequences in string literals
+        r"(((Van|VAN|De|DE|Da|DA)\s)?[A-Z][\w\-'']{2,}(\s(IV|I|II|III|V|Jr\.|JR\.|Sr\.|SR\.))?),",
```

Comment:
This seems fine. Do we have any examples where this would make a difference @legaltextai